apps/editors/tests/test_helpers.py

```diff
@@ -29,7 +29,7 @@
 
 REVIEW_ADDON_STATUSES = (amo.STATUS_NOMINATED, amo.STATUS_LITE_AND_NOMINATED,
                          amo.STATUS_UNREVIEWED)
-REVIEW_FILES_STATUSES = (amo.STATUS_BETA, amo.STATUS_NULL, amo.STATUS_PUBLIC,
+REVIEW_FILES_STATUSES = (amo.STATUS_PUBLIC,
```

Comment:
Some of those tests, in this file, are actually testing weird/impossible scenarios, systematically. I've update them to still pass, but to be sure we cover all the real life scenarios, I've added a new test file `test_review_scenarios.py` to test them.